app/components/explorer/modify-tables-menu.js

```diff
@@ -12,4 +12,8 @@ export default class ModifyTablesMenuComponent extends Component {
   @action closeMenu() {
     this.open = false;
   }
+
+  get acsDataSource() {
```

Comment:
General convention we've followed is to prefix booleans with `is`. For example, `get isSourceAcs()`.